app/src/main/java/org/sopt/seeeuni_30th_homework/SignInActivity.kt

```diff
@@ -0,0 +1,46 @@
+package org.sopt.seeeuni_30th_homework
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import org.sopt.seeeuni_30th_homework.databinding.ActivitySigninBinding
+
+class SignInActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySigninBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySigninBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        initClickEvent()
+    }
+
+    private fun isInputComplete(): Boolean {
+        val id = binding.idEditText.text
+        val pw = binding.pwEditText.text
+        val isIdNull = id.isBlank()
+        val isPwNull = pw.isBlank()
+
+        return !isIdNull && !isPwNull
```

Comment:
```suggestion

        return !(id.isBlank() || pw.isBlank())
```

`isIdNull`, `isPwNull` 변수명을 보면 null값을 검사할 것 같지만 `isBlank()`를 검사하고 있어서 헷갈릴 수도 있을 것 같아요 👀
 변수 없이 이렇게 바꿔보면 어떨까요!😊

<br>

> `isBlank()` : 빈문자열 또는 공백으로만 이루어져 있는지 확인